dev_tools/conf/.pylintrc

```diff
@@ -30,7 +30,6 @@ enable=
     missing-raises-doc,
     mixed-indentation,
     mixed-line-endings,
-    not-callable,
```

Comment:
Why was this removed ?

Reply:
I think this is better handled by mypy, because it's an actual type-checker that can tell if something is callable. pylint relies on heuristics to figure out if something is callable and was producing false positives (@Strilanc filed an issue about this upstream: https://github.com/PyCQA/pylint/issues/2562).